packages/core/src/payment/strategies/paypal-commerce/paypal-commerce-sdk.ts

```diff
@@ -237,9 +389,10 @@ export interface PaypalCommerceInitializationData {
     clientToken?: string;
     attributionId?: string;
     isVenmoEnabled?: boolean;
+    isHostedCheckoutEnabled?: boolean;
 }
 
-export type ComponentsScriptType = Array<'buttons' | 'messages' | 'hosted-fields' | 'payment-fields'>;
+export type ComponentsScriptType = Array<'buttons' | 'messages' | 'hosted-fields' | 'fields'>;
```

Comment:
What difference between `payment-fields` and `fields`? And why did you change it?

Reply:
it is bad merge